### ISSUE TITLE
Windows support is bitrotting

### ISSUE THREAD

**Issue by yanavlasov** [envoyproxy/envoy#28588]
Envoy has lost all Windows maintainers. Envoy Windows build is slowly bitrotting with failing tests just being disabled. Windows toolchain has not been updated for some time and currently blocks update of several dependencies (i.e. CEL) or transition to C++20.

**Comment by yanavlasov** [comment]
CC @envoyproxy/envoy-maintainers

**Comment by phlax** [comment]
+1 the windows ci is now the most flakey part of ci, and is hard to optimize or debug

more often than not resolving windows issues is the blocker to updating/fixing stuff, so mho is that we either need someone who can resolve these issues in a timely fashion or we should drop support

cc @envoyproxy/windows-dev

**Comment by sunjayBhatia** [comment]
Hey all 👋🏽 I'm no longer working on this area at VMware unfortunately, I'll ask around though to see if we can get some more resources

We should probably reassess the members of the windows-dev github team to see who is still active/relevant

**Comment by davinci26** [comment]
Hey same, I am no longer working at Microsoft and I am in the group of folks without access to a Windows machine :( I don't think I can actively help on that front right now

Two cents is that we should flag this to msft folks and maybe a team there has the capacity/motivation to support

**Comment by akshaysngupta** [comment]
Updating here - me and some other folks at msft are helping with keeping windows ci healthy. Dependencies and lack of caching in windows ci are the main pain points.

**Comment by dimo414** [comment]
@akshaysngupta what's the status here? https://github.com/envoyproxy/envoy/issues/33751#issuecomment-2074593048 suggests Windows has in fact been abandoned?

**Comment by dimo414** [comment]
It sounds like it was decided to end Windows support as scheduled despite @akshaysngupta's efforts, @phlax should this issue be updated and closed?

**Comment by jmarantz** [comment]
Just to clarify -- I don't think anyone in the Envoy community objects to having Envoy work on Windows; but the overhead and cost with running the CI workload as part of the normal github flow was too high, and there wasn't enough support or consistent engagement from the windows community to make it go smoothly.

**Comment by dimo414** [comment]
Yes understood, and sorry if I was coming across as critical or judgemental. I appreciate the difficulty of supporting Windows well, and choosing to sunset providing that support instead of letting it drag along makes sense. I'm just trying to follow along with the decisions and current state, since as of Aug. 31st the implication in the comments of this bug was that Windows support _wasn't_ being sunsetted after all.
